docs/develop/go/service-communication.mdx

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 2
+description: "Find out how Restate services can send requests to each other."
+---
+
+import Admonition from '@theme/Admonition';
+
+# Service Communication
+
+A handler can call another handler and wait for the response (request-response), or it can send a message without waiting for the response.
+
+## Request-response calls
+
+Request-response calls are requests where the client waits for the response.
+
+<CH.Scrollycoding className={"single-item"}>
```

Comment:
there is a problem with tab here, not sure what

Reply:
not sure what you mean

Reply:
Look here https://golang.documentation-beg.pages.dev/develop/go/service-communication#request-response-calls